functions/uploads.py

```diff
@@ -133,3 +147,20 @@ def _add_artifact_to_metadata(
     )
 
     return updated_trial_metadata, artifact_metadata
+
+
+def _get_blob_metadata(bucket_name: str, object_name: str):
```

Comment:
I think it will behoove us to put that in the API. So we can use it for manifest xlsx [here](https://github.com/CIMAC-CIDC/cidc-api-gae/pull/97/commits/a7c162c252e9d643f515de5df621fb1fbc60116e#diff-849cdd5351d912f80d3a3d2c206826f0L196) as well

Reply:
Oh, good point.